ffmpeg/api_test.go

```diff
@@ -403,6 +403,7 @@ func TestTranscoder_API_AlternatingTimestamps(t *testing.T) {
 	// tests that also check things like SSIM, MD5 hashes, etc...
 	// See TestNvidia_API_MixedOutput / TestTranscoder_EncoderOpts / TestTranscoder_StreamCopy / TestNvidia_API_AlternatingTimestamps
 	run, dir := setupTest(t)
+	defer os.RemoveAll(dir)
```

Comment:
Oops. At some point, it might be nice to switch to go 1.14 and make use of the `test.Cleanup` helper which can be invoked within `setupTest`, so we don't have to worry about this bit of boilerplate everywhere. See https://www.gopherguides.com/articles/test-cleanup-in-go-1-14/